UPGRADE.rst

```diff
@@ -99,6 +141,35 @@ Synapse will expect these files to exist inside the configured template director
 default templates, see `synapse/res/templates
 <https://github.com/matrix-org/synapse/tree/master/synapse/res/templates>`_.
 
+SMS
+---
+
+Following upgrade, the only way to maintain the ability to register via a phone
+number will be to continue to delegate SMS delivery via the matrix.org and
+vector.im identity servers.
+
+The ``account_threepid_delegates`` dictionary defines whether the
+homeserver should delegate an external server (typically an `identity server
+<https://matrix.org/docs/spec/identity_service/r0.2.1>`_) to handle sending password reset or
+registration messages via email and SMS.
+
+So to delegate SMS sending set ``account_threepid_delegates.sms`` to a base URL of
```

Comment:
```suggestion
So to delegate SMS sending set ``account_threepid_delegates.msisdn`` to a base URL of
```

msisdn means nothing to me. (Well, before I looked it up.) It's a bit surprising to have that as the SMS setting if you're not familiar.

Reply:
I'd prefer not to change the pre-existing setting as part of such a big change. I guess that those unfamiliar will first look in the sample config and then it becomes clear that msisdn is the correct label for phone number

Reply:
nevertheless, as @JorikSchellekens suggests, we need to make the doc match the setting - it should be `msisdn` not `sms` here.